src/styles/cards.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  cursor: pointer;
+  transition: transform $transition-duration;
+
+  &__title {
+    @include main-font-size;
+
+    font-weight: 500;
+    margin: 40px 0 4px;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+    .card__title {
+      color: #34568b;
+    }
+  }
+
+  // &__container {
+  //   display: grid;
+  //   grid-template-columns: 200px;
+  //   gap: 46px 48px;
+  //   justify-content: center;
+  //   padding: 50px 40px;
+
+  //   @media (min-width: 488px) {
+  //     grid-template-columns: 200px 200px;
+  //   }
+
+  //   @media (min-width: 768px) {
+  //     grid-template-columns: repeat(3, 200px);
+  //   }
+
+  //   @media (min-width: 1024px) {
+  //     grid-template-columns: repeat(4, 200px);
+  //   }
+  // }
```

Comment:
please remove the comments if you don't need it

Reply:
yeah, I forgot to do it, thanks